docs/stylesheets/extra.css

```diff
@@ -24,9 +23,9 @@ h1 {
 */
 
 [data-md-color-scheme="slate"] {
-  --md-default-bg-color: rgb(28, 28, 28); /* set sitewide background color*/
-  --md-typeset-color: #cecac3; /* set sitewide text color */
+  --md-default-bg-color: rgb(28, 28, 28); /* set site wide background color*/
+  --md-typeset-color: #cecac3; /* set site wide text color */
   --md-code-bg-color: rgb(20, 20, 23); /* set code-blocks background color*/
   --md-footer-bg-color: rgb(24, 24, 24); /* set footer background color */
-  /* --md-typeset-a-color: #51c0f5; */
+  --md-typeset-a-color: rgb(233, 235, 252); /* set active and link font color */
```

Comment:
Nicely done. There is still an issue with links which are into admonition parts. e.g.:
![Capture d’écran 2021-04-07 à 10 47 00](https://user-images.githubusercontent.com/11842328/113886359-9c65ce80-978e-11eb-95ec-d7f88c0401e6.png)
![Capture d’écran 2021-04-07 à 10 47 02](https://user-images.githubusercontent.com/11842328/113886361-9cfe6500-978e-11eb-8809-c3596377a313.png)
This can be done here or in a different PR.

Reply:
This PR is big enough, so that I will handle this in a different one.